db/migrate/20160218001400_add_default_follow_up_layout_id.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultFollowUpLayoutId < ActiveRecord::Migration
+  def change
+    add_reference :liquid_layouts, :default_follow_up_layout
```

Comment:
What'd be the reasoning behind indexing? I don't see us ever having a large number of layouts or default follow up layouts (maybe like 15-20 max). Can you explain me further the thinking behind your question? I'm assuming there might be something I don't know about the `add_reference` migration.